runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/ServerFactory.java

```diff
@@ -72,6 +73,14 @@ public abstract Server allocatePortAndCreate(
   public abstract Server create(
```

Comment:
Remove this variant and update the allocatePortAndCreate to also take a list of services.

Reply:
Fixed in commit https://github.com/apache/beam/pull/7168/commits/d97bd464982d8aaed0fe05f28ffd40d55ec42e55